packages/pocket-ic/BUILD.bazel

```diff
@@ -66,53 +67,54 @@ rust_test(
     deps = [":pocket-ic"] + DEPENDENCIES + TEST_DEPENDENCIES,
 )
 
-rust_test_suite(
+test_using_pocket_ic_server(
     name = "test",
     size = "small",
     srcs = ["tests/tests.rs"],
     data = [
         "tests/counter.wasm",
         "tests/icp_ledger.wasm",
         ":test_canister.wasm",
-        "//rs/pocket_ic_server:pocket-ic-server",
     ],
     env = {
-        "POCKET_IC_BIN": "$(rootpath //rs/pocket_ic_server:pocket-ic-server)",
         "COUNTER_WASM": "packages/pocket-ic/tests/counter.wasm",
         "LEDGER_WASM": "packages/pocket-ic/tests/icp_ledger.wasm",
         "TEST_WASM": "$(rootpath :test_canister.wasm)",
     },
+    # Override the default ["manual"] extra_HEAD_tags to ensure this test runs on CI.
+    extra_HEAD_tags = [],
+    # Override the default [] extra_mainnet_tags to skip this test on CI.
+    extra_mainnet_tags = ["manual"],
     flaky = False,
+    macro = rust_test_suite,
     proc_macro_deps = MACRO_DEPENDENCIES,
     deps = [":pocket-ic"] + DEPENDENCIES + TEST_DEPENDENCIES,
 )
 
-rust_test_suite(
+test_using_pocket_ic_server(
     name = "restart",
     size = "medium",
     srcs = ["tests/restart.rs"],
-    data = [
-        "//rs/pocket_ic_server:pocket-ic-server",
-    ],
-    env = {
-        "POCKET_IC_BIN": "$(rootpath //rs/pocket_ic_server:pocket-ic-server)",
-    },
+    # Override the default ["manual"] extra_HEAD_tags to ensure this test runs on CI.
+    extra_HEAD_tags = [],
+    # Override the default [] extra_mainnet_tags to skip this test on CI.
+    extra_mainnet_tags = ["manual"],
     flaky = False,
+    macro = rust_test_suite,
     proc_macro_deps = MACRO_DEPENDENCIES,
     deps = [":pocket-ic"] + DEPENDENCIES + TEST_DEPENDENCIES,
 )
 
-rust_test_suite(
+test_using_pocket_ic_server(
     name = "slow",
     size = "medium",
     srcs = ["tests/slow.rs"],
-    data = [
-        "//rs/pocket_ic_server:pocket-ic-server",
-    ],
-    env = {
-        "POCKET_IC_BIN": "$(rootpath //rs/pocket_ic_server:pocket-ic-server)",
-    },
+    # Override the default ["manual"] extra_HEAD_tags to ensure this test runs on CI.
+    extra_HEAD_tags = [],
```

Comment:
1. Does the comment still make sense? It says `default ["manual"]` but I see `default []`. 
2. Having manual as the default could cause some problems during code reviews (a reviewer of a new feature might assume it's sufficiently tested, while the tests might not be activated on CI).